GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/BasicFileSystemTests.cs

```diff
@@ -918,6 +919,27 @@ public void CreateDirectoryInheritsParentDirectoryAttributes(string parentFolder
             FileSystemRunner.DefaultRunner.DeleteDirectory(parentDirectoryPath);
         }
 
+        [TestCase]
+        [Category(Categories.MacOnly)]
+        public void RunPythonExecutable()
+        {
+            GitProcess.Invoke(this.Enlistment.RepoRoot, "checkout FunctionalTests/PythonExecutable");
+
+            string pythonDirectory = Path.Combine(this.Enlistment.RepoRoot, "Test_Executable");
+            string pythonExecutable = Path.Combine(pythonDirectory, "python_wrapper.sh");
+
+            ProcessStartInfo startInfo = new ProcessStartInfo(pythonExecutable);
+            startInfo.RedirectStandardOutput = true;
+            startInfo.RedirectStandardError = true;
+            startInfo.WorkingDirectory = pythonDirectory;

```

Comment:
Do we need to set any of the other properties like `CreateNoWindow` or `UseShellExecute`?  Not sure those make any difference in this test.